frontend/src/metabase/browse/components/BrowseModels.tsx

```diff
@@ -29,7 +29,7 @@ export const BrowseModels = ({
     getCollectionViewPreferences,
   );
 
-  if (error || isLoading) {
+  if (isLoading || error) {
```

Comment:
A trivial change just so there is a diff and I can create a PR separate from the one for `collection-names`